README.md

```diff
@@ -46,20 +46,36 @@ Open VirtualBox, navigate to the settings of your default docker VM, and go to t
 
 ### Running OpenDC
 
-To build and run the full OpenDC stack locally on Linux or Mac, run the commands bellow. Replace `your-google-oauth-client-id` with your `client_id` from the OAuth 2.0 client ID you created, and replace `your-google-oauth-secret` with your `client_secret`.
+To build and run the full OpenDC stack locally on Linux or Mac, you first need to clone the project:
 
 ```bash
 # Clone the repo and its submodules
 git clone --recursive https://github.com/atlarge-research/opendc.git
 
 # Enter the directory
 cd opendc/
+```
+
+In the directory you just entered, you need to set up a small configuration file. To do this, create a file called `keys.json` in the `opendc` folder. In this file, simply replace `your-google-oauth-client-id` with your `client_id` from the OAuth client ID you created.
+
+```json
+{
+  "OAUTH_CLIENT_ID": "your-google-oauth-client-id",
+  "SERVER_BASE_URL": "http://localhost:8081",
+  "ROOT_DIR": "/opendc",
+  "FLASK_SECRET": "This is a super duper secret flask key",
+  "DATABASE_LOCATION": "/data/database/opendc.db"
+}
+```
+
+Once this configuration file is set up, we're ready to get the install up and running:
```

Comment:
To conform with the writing style in the rest of the `README`, suggested rephrase to: "Now, start the server:"

(I don't think the first dependent clause is necessary.)